workspace/da7580f3700dda39fd020841a40c04da/signac_statepoint.json

```diff
@@ -0,0 +1,4 @@
+{
+  "benchmark": "microsphere",
```

Comment:
Do you need to save these in the repo? Can you just have functions that generate these statepoints?